src/Game/Game.elm

```diff
@@ -232,7 +232,7 @@ viewStats : Model -> Html Msg
 viewStats ({ hero } as model) =
     div []
         [ div [] [ text "Stats:" ]
-        , div [] [ text <| "HP: " ++ (Stats.printHP hero.stats) ]
+        , div [ class (Stats.getHPClass hero.stats) ] [ text <| "HP: " ++ (Stats.printHP hero.stats) ]
```

Comment:
I tried to move it but run into "cyclic dependencies" problem. Check out the last commit to see the other changes I've made.